src/components/SidePanel/SidePanel.stories.tsx

```diff
@@ -255,3 +253,17 @@ export const Wide: Story = {
   },
   render: StoryExample,
 };
+
+export const Animated: Story = {
+  args: {
+    className: "",
+    hasError: false,
+    parentId: "l-application-default",
+    pinned: false,
+    loading: false,
+    overlay: true,
+    width: "",
+    isAnimated: true,
+  },
+  render: StoryExample,
+};
```

Comment:
I think there is the expectation of the caller to have `overflow:hidden` on the document body, so this doesn't come with a scrollbar during the animation. Though, I am not really sure how and where to put that information. Maybe on the is animated prop doc block?